TEP008.rst

```diff
@@ -0,0 +1,46 @@
+TEP008: Tighter integration of Atomic Data, Plasma and Model
+============================================================
+
+Status
+======
+**Discussion**
+
+Responsible
+===========
+
+@wkerzendorf
+@yeganer
+
+Branches and Pull requests
+==========================
+
+Description
+===========
+
+TARDIS current plasma is structured in a very modular way that allows easy
+testing and quick addition of new plasma modules. We restructured the Plasma
+to this new form in 2015 and implemented the most important parts of the
+framework. Due to time constraints, not all functionality was moved to this new
+structure and there is still considerable amount of legacy code. In addition,
+while the functionality is there we lack documentation that explains the new
+rationale to the collaboration and users.
+
+With this TEP we have multiple aims - ensure that all remaining parts that will
```

Comment:
I think we have to differentiate between the goals and the steps to achieve them. That was a bit misleading by me.
In my opinion the goal of this TEP should be to have all the information that defines the model connected through networkx and stored in a graph that is associated with the model (instead of the plasma)
To achieve that in a 3 month window, implemented by someone unfamiliar with the codebase is a huge challenge. Therefore it might be reasonable to have intermediate goals. These are kind of listed in my comment about the Implementation part.

As for explaining. I'd start by saying that we work with a complex model that has the advantage that information is only flowing in one direction(because we are iterating) and that we can draw the relationship between Properties in a graph. In our Implementation we mirror this by creating a graph with networkx which is used to track the information-flow. This idea was tested with the plasma part and because it worked well the rest of the code should be propagated to this new structure.

I think you could copy that last part and modify it slightly and it should be okay for now.